docs/runtime_options.md

```diff
@@ -152,6 +152,9 @@ Core perf events can be exposed on Prometheus endpoint per CPU or aggregated by
 - `--disable_metrics="percpu"` - core perf events are aggregated
 - `--disable_metrics=""` - core perf events are exposed per CPU.
 
+It's possible to get "too many opened files" error when a lof of perf events are exposed per CPU. This happens because of passing system limits.
+Try to increase it with `ulimit -n <value>`.
```

Comment:
Can you note that `ulimit -n <value>` increases max number of file descriptions?

Reply:
Is it good now?